keyvault/spring-cloud-azure-starter-keyvault-secrets/multiple-property-source/src/main/resources/application.yml

```diff
@@ -8,16 +8,6 @@ spring:
             -
               name: key-vault-1
               endpoint: ${ENDPOINT_1}
-              profile:
-                tenant-id: ${AZURE_TENANT_ID_1}
-              credential:
-                client-id: ${AZURE_CLIENT_ID_1}
-                client-secret: ${AZURE_CLIENT_SECRET_1}
             -
               name: key-vault-2
               endpoint: ${ENDPOINT_2}
-              profile:
-                tenant-id: ${AZURE_TENANT_ID_2}
-              credential:
-                client-id: ${AZURE_CLIENT_ID_2}
-                client-secret: ${AZURE_CLIENT_SECRET_2}
```

Comment:
Did you run this sample? Will it work if these contents are deleted?

Reply:
sure